third_party/nvidia/hopper/lib/Transforms/WarpSpecialization/WSSpecialize.cpp

```diff
@@ -194,9 +194,11 @@ Operation *SpecializeIfOp(scf::IfOp ifOp, IRMapping &mapping,
   for (auto idx : keptResultVec) {
     newResultTypes.push_back(ifOp->getResultTypes()[idx]);
   }
+  builder.setLoopScheduleInfo(ifOp);
```

Comment:
These should be no-ops but I added them for consistency. Technically the IF can be marked as a prologue/epilogue, so its probably good to include.